teamworkApp/lib/dbCalls.py

```diff
@@ -123,38 +122,43 @@ def get_all_student_IDs(test=False):
 
 def insert_student_team_pairs(student_and_team_ids, test=False):
     """
-    Insert list of student_id team_id pairs into database
+    Insert a list of (student_id, team_id pairs) into database
```

Comment:
I would put pairs outside of the brackets if you leave the word in the comments.